### DISCUSSION TITLE
Add inline relation marks (for internal linking)

### DISCUSSION THREAD

**Discussion by formfcw** [formfcw/directus-extension-flexible-editor#1] [Feature Request]
In addition to `relation blocks` it would be nice to integrate `relation marks`. This would cover the use case for internal linking.

A user could select text and then relate to a M2A item.

**Comment by jdjoshuadavison** [comment]
Was about to submit a new feature request for this exact functionality.

**Comment by erichjsonfosse** [comment]
This would be very nice, yes.

@formfcw Let me know if there's anything I can do to help!

**Reply by formfcw** [reply]
Thank you for your offer. I am planning a complete rewrite of the relationship thing. I may let you know when I've done the groundwork and I am ready to accept contributions for this feature.

**Reply by geekyayush** [reply]
> Thank you for your offer. I am planning a complete rewrite of the relationship thing. I may let you know when I've done the groundwork and I am ready to accept contributions for this feature.

Hi,
Just checking in. Is this done?
Also, let me know if I can be of any help.
Thanks.

**Reply by formfcw** [reply]
Hello there! Thank you for your interest. It is not ready yet. I am actively working on it. I might let you know if you can help me.
